openedx/adg/lms/applications/managers.py

```diff
@@ -23,29 +23,47 @@ class MultilingualCourseGroupManager(Manager):
     Manager for MultilingualCourseGroup
     """
 
+    def common_course_groups(self):
+        return self.get_queryset().filter(
+            is_common_business_line_prerequisite=False, business_line_prerequisite__isnull=True
+        )
+
     def prereq_course_groups(self):
         """
         Get non-empty prerequisite course groups
         """
         return self.get_queryset().filter(is_program_prerequisite=True, multilingual_courses__isnull=False).distinct()
 
-    def get_courses(self, user, is_prereq=False):
+    def business_line_course_groups(self, user):
+        return self.get_queryset().filter(
+            Q(is_common_business_line_prerequisite=True) | Q(business_line_prerequisite=user.application.business_line)
+        )
+
+    def get_courses(self, user, is_prereq=False, business_line_courses=False):
```

Comment:
@asadali145 here are terminologies we are currently using for different kind of pr reqs
- Program pre reqs (previously known as per reqs)
- Business line pre req (pre reqs of a specific business line)
- Common business line pre req (Pre reqs which are common for all of the business lines)

Let's please update our variable names at all of the places accordingly, in order to enhance the readability.